sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -62,6 +62,20 @@ class MyDialect extends DefaultParserDialect
  * valid, but Hive currently cannot execute it.
  */
 class SQLQuerySuite extends QueryTest {
+
+  /**
+   * Drops table `tableName` after calling `f`.
+   * This is the `SQLTestUtils` function without
+   * an explicit sql context
```

Comment:
Why not just mix in `SQLTestUtils` then implement/override its `sqlContext` method to return `TestHive`?